control/tests/statefbk_test.py

```diff
@@ -157,6 +157,29 @@ def testAcker(self):
                 np.testing.assert_array_almost_equal(np.sort(poles),
                                                      np.sort(placed), decimal=4)
 
+    def testPlace(self):
+        # Matrices shamelessly stolen from scipy example code.
+        A = np.array([[1.380,  -0.2077,  6.715, -5.676],
+                      [-0.5814, -4.290,   0,      0.6750],
+                      [1.067,   4.273,  -6.654,  5.893],
+                      [0.0480,  4.273,   1.343, -2.104]])
+
+        B = np.array([[0,      5.679],
+                      [1.136,  1.136],
+                      [0,      0,],
+                      [-3.146,  0]])
+        P = np.array([-0.5+1j, -0.5-1j, -5.0566, -8.6659])
+        K = place(A, B, P)
```

Comment:
Can we also add some tests for `place_varga` and `acker` here, as a way to increase coverage?

Reply:
I think `acker` already has a test [(line 125)](https://github.com/python-control/python-control/blob/df4ee828ec443a816c52285bc8df2589706ee132/control/tests/statefbk_test.py#L125)...

I added a test for place_varga.